great_expectations/datasource/fluent/pandas_datasource.py

```diff
@@ -657,9 +656,9 @@ def _validate_asset_name(asset_name: Optional[str] = None) -> str:
             asset_name = DEFAULT_PANDAS_DATA_ASSET_NAME
         return asset_name
 
-    def _get_validator(
+    def _get_batch(
```

Comment:
All the functional changes are in the file. The other changes are tests/docs/pyi related.